dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java

```diff
@@ -102,6 +103,35 @@ public Organisation approveOrganisation(@ApiParam(hidden = true) @Auth User user
         return organisationDAO.findById(id);
     }
 
+    @POST
+    @Timed
+    @UnitOfWork
+    @RolesAllowed({ "curator", "admin" })
+    @Path("{organisationId}/reject/")
+    @ApiOperation(value = "Rejects an organisation.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Admin/curator only", response = Organisation.class)
+    public Organisation rejectOrganisation(@ApiParam(hidden = true) @Auth User user,
+            @ApiParam(value = "Organisation ID.", required = true) @PathParam("organisationId") Long id) {
+        Organisation organisation = organisationDAO.findById(id);
+        if (organisation == null) {
+            String msg = "Organisation not found";
+            LOG.info(msg);
+            throw new CustomWebApplicationException(msg, HttpStatus.SC_NOT_FOUND);
+        }
+
+        if (!Objects.equals(organisation.getStatus(), Organisation.ApplicationState.APPROVED)) {
```

Comment:
1. I wonder if you shouldn't allow rejecting of an approved organization. What if some org inadvertently or incorrectly gets approved?
2. If you decide to keep it this way, I think you should have an else for this if and return a 400 bad request.